### DISCUSSION TITLE
Datetime log or message on fare check

### DISCUSSION THREAD

**Discussion by joeycf** [jdholtz/auto-southwest-check-in#196] [Ideas]
I was trying to mess around with the checking intervals and I wanted to implement an output of a console log for every time the autochecker "checks" for the fare. I was wondering if someone could point me in the right direction as I couldn't seem to get it working for me.

**Comment by jdholtz** [comment]
Running the script with the `--verbose` flag will give you more information on the script checking for fares. It will happen right after flights are scheduled and every X hours after (based on the retrieval interval set in the config).

To implement a print statement manually, you can add `print("Checking fares")` on line 93 of `lib/reservation_monitor.py`.

**Reply by joeycf** [reply]
Oh perfect thank you!
